.github/workflows/ci.yml

```diff
@@ -10,10 +10,14 @@ on:
   pull_request:
     branches:
       - main
+    types: [opened, ready_for_review, synchronize]
 
 jobs:
   tests:
     runs-on: ubuntu-latest
+    if: |
+        !contains(github.event.head_commit.message, '[SKIP TESTS]') &&
```

Comment:
I don't think we need this `skip tests` stuff, since tests are quite fast in Solar :)